### DISCUSSION TITLE
Planner with sans serif fonts

### DISCUSSION THREAD

**Discussion by nsturko** [kudrykv/latex-yearly-planner#93] [General]
Hello,

first of all, I would like to express gratitude for this great project!

Second, since I am nowhere close to being Latex proficient, would anyone be willing to compile the planner with sans serif font (without strokes as in serif font family)?

Thank you in advance!

![image](https://github.com/kudrykv/latex-yearly-planner/assets/60796036/cf9dc8cd-554e-46eb-84ab-d31bd5677ab5)

**Comment by Sterh20** [comment]
What kind of font does the planer use now? In Adobe Acrobat it says "CM R 9".

**Comment by bioluminesceme** [comment]
If you use xelatex instead of pdflatex you can use custom fonts. I have MiKTeX and the 'xetex' and 'fontspec' packages seems to be enabled by default

in tpls/document.tpl add fontspec to the usepackage list and set the main font to something you have installed, the name has to be precise. On windows, click start, search for "Fonts" and it should open Control Panel\Appearance and Personalization\Fonts\

Open the font you want to use, and check what Font name it has
![fontname](https://github.com/kudrykv/latex-yearly-planner/assets/9899593/672d6f8d-bb45-48cd-97d0-eee66d0b6cbe)



    \usepackage{multido} #this is the current existing last package in the list
    \usepackage{fontspec}

    \setmainfont{Open Sans Medium} 
    
    

Normally you would run 

    pdflatex rm2.mos.default.dailycal.tex
    
Now you run 

    xelatex rm2.mos.default.dailycal.tex
    
(if it outputs a PDF where some pages are a bit wonky, just run this command again).

With Open Sans Medium as the default font.

![Screenshot 2023-12-02 124102](https://github.com/kudrykv/latex-yearly-planner/assets/9899593/9a6c90cf-8e78-4ea0-be21-1d9e443ecbe7)

**Reply by nsturko** [reply]
It worked!

Thank you!

Best regards
N

**Comment by nsturko** [comment]
Its interesting how using xelatex instead of pdlatex produces final planners around 3 mb in size, while using pdflatex files are aroun 13mb in size.
Maybe @kudrykv can put some light into this :)

**Reply by kudrykv** [reply]
that's bizarre
`pdflatex` produces PDFs for me that are around 2-3 mb in size
which OS are you running on? do you have the latest latex version?

**Reply by bioluminesceme** [reply]
Much smaller file size for me too with xelatex  vs pdflatex (win11)

$ pdflatex --version
MiKTeX-pdfTeX 4.16 (MiKTeX 23.10.12)
© 1982 D. E. Knuth, © 1996-2023 Hàn Thế Thành
TeX is a trademark of the American Mathematical Society.
using bzip2 version 1.0.8, 13-Jul-2019
compiled with curl version 8.4.0; using libcurl/8.4.0 Schannel
compiled with expat version 2.5; using expat_2.5.0
compiled with jpeg version 9.5
compiled with liblzma version 50040002; using 50040002
compiled with libpng version 1.6.39; using 1.6.39
compiled with libressl version LibreSSL 3.8.1; using LibreSSL 3.8.1
compiled with MiKTeX Application Framework version 4.8; using 4.8
compiled with MiKTeX Core version 4.22; using 4.22
compiled with MiKTeX Archive Extractor version 4.0; using 4.0
compiled with MiKTeX Package Manager version 4.9; using 4.9
compiled with uriparser version 0.9.7
compiled with xpdf version 4.04
compiled with zlib version 1.2.13; using 1.2.13

**Reply by kudrykv** [reply]
Yeah, apparently I have something else
```
❯ pdflatex --version
pdfTeX 3.141592653-2.6-1.40.25 (TeX Live 2023)
kpathsea version 6.3.5
Copyright 2023 Han The Thanh (pdfTeX) et al.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the pdfTeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the pdfTeX source.
Primary author of pdfTeX: Han The Thanh (pdfTeX) et al.
Compiled with libpng 1.6.39; using libpng 1.6.39
Compiled with zlib 1.2.13; using zlib 1.2.13
Compiled with xpdf version 4.04
```

**Reply by nsturko** [reply]
Debian 12
pdfTeX 4.16 (MiKTex 23.10).

Maybe its because you use TeX Live instead of MikTex?

**Reply by kudrykv** [reply]
yep

**Reply by nsturko** [reply]
Ok, installed TeX Live 2023 on Fedora 38, seems that pdflatex now produces files around 3 MB in size.
